model/annotations/task_annotations.go

```diff
@@ -204,6 +204,49 @@ func UpdateAnnotation(a *TaskAnnotation, userDisplayName string) error {
 	return errors.Wrapf(err, "problem adding task annotation for '%s'", a.TaskId)
 }
 
+// InsertManyAnnotations updates the source for a list of task annotations and their issues and then inserts them into the DB.
+func InsertManyAnnotations(annotations []TaskAnnotation, userDisplayName string) error {
+	source := &Source{
+		Author:    userDisplayName,
+		Time:      time.Now(),
+		Requester: APIRequester,
+	}
+	for i, _ := range annotations {
+		update := bson.M{}
```

Comment:
This seems to be reusing a lot of the work done [here](https://github.com/evergreen-ci/evergreen/pull/5561/files). Do we want to maybe extract that into it's own function so that we can reuse whatever code we can?

Reply:
Additionally, there is some conversation there about how the note and metadata should be handled. Please coordinate with that and make sure that the same is done for this route.